SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -2047,11 +2047,44 @@ void ActorSystem::SendCreateEntityRequest(USpatialActorChannel& ActorChannel, ui
 	UE_LOG(LogActorSystem, Log, TEXT("Sending create entity request for %s with EntityId %lld, HasAuthority: %d"), *Actor->GetName(),
 		   ActorChannel.GetEntityId(), Actor->HasAuthority());
 
-	SpatialGDK::EntityFactory DataFactory(NetDriver, NetDriver->PackageMap, NetDriver->ClassInfoManager, NetDriver->RPCService.Get());
-	TArray<FWorkerComponentData> ComponentDatas = DataFactory.CreateEntityComponents(&ActorChannel, OutBytesWritten);
+	EntityFactory DataFactory(NetDriver, NetDriver->PackageMap, NetDriver->ClassInfoManager, NetDriver->RPCService.Get());
+	TArray<FWorkerComponentData> ComponentDatas;
+	TArray<FWorkerComponentUpdate> ComponentUpdates;
+
+	enum EEntityCreationType
+	{
+		CreateNewEntity,
+		PopulateSkeleton,
+	};
+
+	const EntityViewElement* ExistingEntity = NetDriver->Connection->GetCoordinator().GetView().Find(ActorChannel.GetEntityId());
```

Comment:
Can we get this through a subview instead of through the net driver?

Reply:
Sure - just to confirm, it'd be `MySubview->GetView.Find()` as skeletons are incomplete on any existing subviews. Or we could get us an `UnpopulatedSkeletonEntitySubview`...